tests/test-helpers.js

```diff
@@ -7,7 +7,24 @@ import skipInPhantom from './helpers/skip-in-phantom';
 import MobiledocHelpers from './helpers/mobiledoc';
 import PostAbstract from './helpers/post-abstract';
 
-const { test } = QUnit;
+const { test:qunitTest, module } = QUnit;
+
+QUnit.config.urlConfig.push({
+  id: 'debugTest',
+  label: 'Debug Test'
+});
```

Comment:
this is a little odd, perhaps, but it greatly improves debugging speed for acceptance tests. This adds a checkbox 'DebugTest' in the QUnit toolbar. Clicking it causes the page to reload and drop into the debugger right before starting the test. The flow I've been using is:

  * run all tests (with dev tools open), click "rerun test" on any failing acceptance test
  * click the "Debug Test" checkbox -> page reloads and the debugger activates
  * step into the `originalCallback` test function and step through the lines in the test

I've found this to be a lot quicker than finding the test code in my text editor and reading through the steps to figure out what it is testing and what should be happening.

The caveat is that you need to `import {test, module} from Helpers` in the acceptance test (instead of `import {test, module} from QUnit`)